winpr/libwinpr/sspi/Kerberos/kerberos.c

```diff
@@ -244,7 +126,7 @@ static krb5_error_code krb5_prompter(krb5_context context, void* data, const cha
 
 	for (i = 0; i < num_prompts; i++)
 	{
-		if (ptypes && ptypes[i] == KRB5_PROMPT_TYPE_PREAUTH)
+		if (ptypes && ptypes[i] == KRB5_PROMPT_TYPE_PREAUTH && data)
```

Comment:
Just curious, have you ever hit the case where `data` is NULL ?

Reply:
I don't think so... I don't remember why I put that in although I suppose it doesn't hurt